tools/ci-cdk/canary-runner/src/build_bundle.rs

```diff
@@ -517,7 +523,7 @@ default = ["v0.4.1"]
                     .collect(),
                     release: None,
                 },
-                release_tag: ReleaseTag::from_str("release-2022-07-26").unwrap(),
+                release_tag: ReleaseTag::from_str("release-2023-05-26").unwrap(),
```

Comment:
Should this be `release-2023-01-26`?

Reply:
For this test, it needs to be a release tag _after_ `release-2023-01-26`. The code will see that `release-2023-01-26 < release-2023-05-26`, and thus, add the `release-2023-01-26` feature from the notable releases list.